### PR TITLE
Add formula for calculating SRM

### PR DESCRIPTION
**Pull request by shouptech**
I have added a formula to the Recipe class which will calculate a recipe's color in units SRM.

### CODE REVIEW

File: pybeerxml/recipe.py [hunk 1]
```diff
@@ -92,6 +92,15 @@ def fg(self):
 
         return _fg
 
+    @property
+    def color(self):
+        # Formula source: http://brewwiki.com/index.php/Estimating_Color
+        mcu = 0
+        for f in self.fermentables:
+            mcu += f.amount * f.color * 8.3453 / self.batch_size
```
Comment (hotzenklotz) [inline pybeerxml/recipe.py]: The default values for `amount` and `color` are `None`. ([see here](https://github.com/hotzenklotz/pybeerxml/blob/master/pybeerxml/fermentable.py#L13)) Perhaps we should be a bit more defensive here and check that both properties are not `None`. What do you think?

Comment (shouptech) [reply pybeerxml/recipe.py]: Agreed! I'll update the code.

**Comment by shouptech** [comment]
The formula is correct. `8.3453` is the conversion factor from kg/L to lb/gal. I checked it against BeerSmith and it's within a hundredth of SRM.

File: pybeerxml/recipe.py [hunk 2]
```diff
@@ -92,6 +92,17 @@ def fg(self):
 
         return _fg
 
+    @property
+    def color(self):
+        # Formula source: http://brewwiki.com/index.php/Estimating_Color
+        mcu = 0.0
+        for f in self.fermentables:
+            if f.amount is not None and f.color is not None:
+                # 8.3454 is conversion factor from kg/L to lb/gal
```
Comment (hotzenklotz) [inline pybeerxml/recipe.py]: 👍

**Comment by hotzenklotz** [comment]
@shouptech I released version `1.0.6` on PyPi which includes your PR. Thank you very much.

**Pull request merged**